operations/research-and-development/product/release-process/release-overview.md

```diff
@@ -74,19 +74,25 @@ The goal is not to have the same release cadence for all products, but there may
 
 Currently Mattermost Cloud releases occur on a 2-week cycle, but the goal is to release more frequently. Cloud and self-managed PRs don’t need to be cherry-picked to releases, except for any last minute bug fixes that get merged after the release branch is cut, and any necessary hotfixes.
 
-Schedule for Mattermost Cloud releases:
-
-* \(T-7\): 7 working days prior to the release day, the `master` branch is merged to the Cloud branch.
-* \(T-0\): Release Day
-
-Schedule for self-managed releases:
-
-* \(T-12\): Feature Review/Judgment Day/Release branch cut
-* \(T-11\): RC1 Cut
-* \(T-10\): RC testing and final QA testing
-* \(T-5\): Code Freeze
-* \(T-2\): Cut Final
-* \(T-0\): Release Day
+We follow [the Agile Release Train method](https://www.scaledagileframework.com/agile-release-train/) at least for Cloud releases. If releases are not approved by a certain date, then we miss the release train.
+
+**Schedule for Cloud releases**:
+ - Thursdays: Merge master into the cloud branch and update cloud test servers.
+ - Friday-Tuesday: Release testing and bug fixing.
```

Comment:
I would still like to have Friday for release testing so that Monday/Tuesday can be used for reviewing failures.  Tests take time to run in RF due to having to set up a webhook for each run and it helps that I can run these on Fridays.  So having the cloud test servers updated on Thursdays is also important

Reply:
Thanks Lindy! Then I'm good as is.